CMakeLists.txt

```diff
@@ -236,53 +181,24 @@ endif()
 
 set_project_standards(_CuraEngine)
 enable_sanitizers(_CuraEngine)
-set_rpath(TARGETS
-        _CuraEngine
-        PATHS
-        "$<$<PLATFORM_ID:Linux>:usr/bin>"
-        "$<$<PLATFORM_ID:Linux>:usr/bin/lib>"
-        "$<$<PLATFORM_ID:Darwin>:../lib>"
-        RELATIVE)
 
 if(${EXTENSIVE_WARNINGS})
     set_project_warnings(_CuraEngine)
 endif()
 
-if(NOT APPLE AND ENABLE_OPENMP)
-    find_package(OpenMP REQUIRED)
-    target_link_libraries(_CuraEngine PUBLIC OpenMP::OpenMP_CXX)
+if(ENABLE_OPENMP)
```

Comment:
Yet another sneaky change that would require a lot of extra testing. Yes it should _theoretically_ work, but i would expect a lot of prints need to be done to validate that this works (as it's an actual functional change!)

Reply:
You're right. I will revert this such that we can test it out in an actual ticket. This was me getting over enthusiastic because I found an OpenMP recipe in conan-center-index.